google/cloud/bigquery/integration_tests/bigquery_read_integration_test.cc

```diff
@@ -80,6 +81,19 @@ std::int64_t CountRowsFromStream(
   return num_rows;
 }
 
+template <typename Functor>
+google::cloud::internal::invoke_result_t<Functor> StatusRetryLoop(
+    Functor&& operation,
+    std::vector<int> const& delay_seconds = {10, 30, 60, 60}) {
```

Comment:
Nit: I'd suggest a vector of durations.  That is, convert from `int` as early as possible.